orga/app/templates/components/routes/authenticated/campaigns/details/participants-tab.hbs

```diff
@@ -6,7 +6,7 @@
           <th>Nom</th>
           <th>Prénom</th>
           {{#if @campaign.idPixLabel}}
-            <th>{{@idPixLabelDisplay}}</th>
+            <th>{{this.idPixLabelDisplay}}</th>
```

Comment:
d'où vient `this.idPixLabelDisplay` si `get idPixLabelDisplay()` est supprimé du composant?

Reply:
désolée, ya une partie de mon commit qui s'est fait la malle ^^